solutions/catalog/variables.tf

```diff
@@ -63,3 +63,18 @@ variable "disabled" {
   description = "Denotes whether a catalog is disabled."
   default     = false
 }
+
+variable "target_accounts" {
+  type = list(object({
+    api_key            = optional(string)
+    name               = string
+    label              = string
+    project_id         = optional(string)
+    trusted_profile_id = optional(string)
+    target_service_id  = optional(string)
+  }))
+  default     = []
+  nullable    = false
+  description = "List of target accounts to add to this catalog. Can only be configured on an update, not on a create."
```

Comment:
This is a complex object so users in projects wil not know the required format. Please add a markdown doc with this information and a "Learn more" link in the variable description that links to it. As an example see [here](https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/625586407cf8537b1f24f78f4d394f7f03cc691f/solutions/fully-configurable/variables.tf#L98)